src/observe-tag.js

```diff
@@ -1,6 +1,15 @@
 const { observe } = require('@nx-js/observer-util')
 const { TRAM_TAG_REACTION, TRAM_TAG_NEW_EFFECTS, TRAM_TAG_CLEANUP_EFFECTS } = require('./node-names')
 
+// helper functions for sorting
+const toIndicies = (node, index) => index
+const toNodes = elementAndChildren => index => elementAndChildren[index]
+const byDistanceFromIndex = targetIndex => (indexA, indexB) => {
+	const diffFromTargetA = Math.abs(indexA - targetIndex)
+	const diffFromTargetB = Math.abs(indexB - targetIndex)
+	return diffFromTargetA < diffFromTargetB ? -1 : 1
+}
```

Comment:
this sorting function prioritizes indices that are closest to a target.
E.g. if the target is `5` out of `[1, 2, 3, 4, 5, 6, 7, 8, 9]`, 
then the new sorting would be `[5, 4, 6, 3, 7, 2, 8, 1, 9]`. 

I don't know if there's a more elegant way to write this, but this works :+1: